internal/entity/entity.go

```diff
@@ -0,0 +1,84 @@
+package entity
+
+import (
+	"context"
+
+	"github.com/dogmatiq/configkit"
+)
+
+// Application is an implementation of config.Application.
+type Application struct {
+	IdentityValue configkit.Identity
+	TypeNameValue string
+	Messages      configkit.EntityMessageNames
+	HandlerSet    configkit.HandlerSet
```

Comment:
```suggestion
	MessageNameValue      configkit.EntityMessageNames
	HandlersValue    configkit.HandlerSet
```

Should we just make these all use `[method name] + Value`, like this?

Reply:
Yeah I was thinking about it, but then foolishly made exceptions for `Messages`, etc. Let me change everything to look uniform.

Reply:
Ok that should be done.